src/App.js

```diff
@@ -1,14 +1,25 @@
 import React, { Component } from 'react'
 import { Route, NavLink } from 'react-router-dom'
 import Navigation from './components/Navigation/Navigation'
```

Comment:
;;;;;;;;;;

Reply:
to make this more official, I believe we need to add semi-colons here for consistency. thank you

Reply:
yes ma'am 😅